packages/nux/src/components/guide/page-control.js

```diff
@@ -0,0 +1,33 @@
+/**
+ * External dependencies
+ */
+import { times } from 'lodash';
+
+/**
+ * WordPress dependencies
+ */
+import { __, sprintf } from '@wordpress/i18n';
+import { IconButton } from '@wordpress/components';
+
+/**
+ * Internal dependencies
+ */
+import { PageControlIcon } from './icons';
+
+export default function PageControl( { currentPage, numberOfPages, setCurrentPage } ) {
+	return (
+		<ul className="nux-guide__page-control" aria-label={ __( 'Guide controls' ) }>
```

Comment:
See:
https://github.com/WordPress/gutenberg/blob/853b3cb100469f9d71bad93bb38ece5210dae0b2/packages/edit-post/src/components/keyboard-shortcut-help-modal/index.js#L44-L49

Do we need it here as well? We should finally abstract it away, as it pops up way too often.

Reply:
Will defer to @enriquesanchez here. He said in https://github.com/WordPress/gutenberg/pull/18041#issuecomment-559300177 that this worked in VoiceOver, so I'm not sure that we need to change the markup.

Reply:
It would be great if they fix it.

Reply:
Are you referring to the `ul`?

Reply:
Yes, it was problematic in other places so we had to explicitly provide the `role="list"`.